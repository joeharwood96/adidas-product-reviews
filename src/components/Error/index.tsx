import React from 'react'
import './styles.scss'

import logo from '../../assets/Adidas_Logo.svg'

export default function Error() {

    return (
        <div className="error">
            <div className="error-center">
                <img src={logo} alt="Adidas logo" className="loader-logo" />
                <h2>Something went wrong, please try again later.</h2>
            </div>
        </div>
    )
}