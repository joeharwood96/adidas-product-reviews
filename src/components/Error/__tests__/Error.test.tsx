import React from "react";
import Error from "../index";

describe("<Error />", () => {
    test("should display the Error", () => {
        const component = <Error />
        expect(component).toMatchSnapshot()
    });
});