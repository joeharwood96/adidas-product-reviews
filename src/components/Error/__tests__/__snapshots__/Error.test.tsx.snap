// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should display the Error 1`] = `<Error />`;
